Makefile

```diff
@@ -116,188 +116,25 @@ start-docker: ## Starts the docker containers for local development.
-		echo restarting mattermost-postgres; \
-		docker start mattermost-postgres > /dev/null; \
-	fi
-
-	@if [ $(shell docker ps -a --no-trunc --quiet --filter name=^/mattermost-inbucket$$ | wc -l) -eq 0 ]; then \
-		echo starting mattermost-inbucket; \
-		docker run --name mattermost-inbucket -p 9000:10080 -p 2500:10025 -d jhillyerd/inbucket:release-1.2.0 > /dev/null; \
-	elif [ $(shell docker ps --no-trunc --quiet --filter name=^/mattermost-inbucket$$ | wc -l) -eq 0 ]; then \
-		echo restarting mattermost-inbucket; \
-		docker start mattermost-inbucket > /dev/null; \
-	fi
-
-	@if [ $(shell docker ps -a --no-trunc --quiet --filter name=^/mattermost-minio$$ | wc -l) -eq 0 ]; then \
-		echo starting mattermost-minio; \
-		docker run --name mattermost-minio -p 9001:9000 -e "MINIO_ACCESS_KEY=minioaccesskey" \
-		-e "MINIO_SSE_MASTER_KEY=my-minio-key:6368616e676520746869732070617373776f726420746f206120736563726574" \
-		-e "MINIO_SECRET_KEY=miniosecretkey" -d minio/minio:RELEASE.2019-04-23T23-50-36Z server /data > /dev/null; \
-		docker exec -it mattermost-minio /bin/sh -c "mkdir -p /data/mattermost-test" > /dev/null; \
-	elif [ $(shell docker ps --no-trunc --quiet --filter name=^/mattermost-minio$$ | wc -l) -eq 0 ]; then \
-		echo restarting mattermost-minio; \
-		docker start mattermost-minio > /dev/null; \
-	fi
-
-ifeq ($(BUILD_ENTERPRISE_READY),true)
-	@echo Ldap test user test.one
-	@if [ $(shell docker ps -a --no-trunc --quiet --filter name=^/mattermost-openldap$$ | wc -l) -eq 0 ]; then \
-		echo starting mattermost-openldap; \
-		docker run --name mattermost-openldap -p 389:389 -p 636:636 \
-			-e LDAP_TLS_VERIFY_CLIENT="never" \
-			-e LDAP_ORGANISATION="Mattermost Test" \
-			-e LDAP_DOMAIN="mm.test.com" \
-			-e LDAP_ADMIN_PASSWORD="mostest" \
-			-d osixia/openldap:1.2.2 > /dev/null;\
-		sleep 10; \
-		docker cp tests/test-data.ldif mattermost-openldap:/test-data.ldif;\
-		docker cp tests/qa-data.ldif mattermost-openldap:/qa-data.ldif;\
-		docker exec -ti mattermost-openldap bash -c 'ldapadd -x -D "cn=admin,dc=mm,dc=test,dc=com" -w mostest -f /$(LDAP_DATA)-data.ldif';\
-	elif [ $(shell docker ps | grep -ci mattermost-openldap) -eq 0 ]; then \
-		echo restarting mattermost-openldap; \
-		docker start mattermost-openldap > /dev/null; \
-		sleep 10; \
-	fi
-
-	@if [ $(shell docker ps -a --no-trunc --quiet --filter name=^/mattermost-elasticsearch$$ | wc -l) -eq 0 ]; then \
-		echo starting mattermost-elasticsearch; \
-		docker run --name mattermost-elasticsearch -p 9200:9200 -e "http.host=0.0.0.0" -e "transport.host=127.0.0.1" -e "ES_JAVA_OPTS=-Xms250m -Xmx250m" -d mattermost/mattermost-elasticsearch-docker:6.5.1 > /dev/null; \
-	elif [ $(shell docker ps --no-trunc --quiet --filter name=^/mattermost-elasticsearch$$ | wc -l) -eq 0 ]; then \
-		echo restarting mattermost-elasticsearch; \
-		docker start mattermost-elasticsearch> /dev/null; \
-	fi
+	docker-compose --no-ansi -f ./build/docker-compose.yml run --rm start_dependencies
```

Comment:
We have removed the already running checks. Does it work fine if run multiple times?

Reply:
@crspeller `make start-docker` runs fine when run multiple times.  The containers are checked if they are already running, and if not starts them.  Was this your question?

Reply:
Yup. As long as it can be run multiple times it should be fine.